### PR TITLE
[docker] retry apt download at most 3 times

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

**Comment by jigold** [comment]
I think this is what Dania ran into with the six package:
https://ci.hail.is/batches/131308/jobs/94

File: benchmark/Dockerfile [hunk 1]
```diff
@@ -2,8 +2,8 @@ FROM ubuntu:18.04
 
 ENV LANG C.UTF-8
 
-
-RUN apt-get update && \
+RUN echo "APT::Acquire::Retries \"3\";" > /etc/apt/apt.conf.d/80-retries && \
+  apt-get update && \
```
Comment (jigold) [inline benchmark/Dockerfile:6]: Some have apt-get update -y but others don't. Should we make it consistent?

**Review by jigold — DISMISSED** [review]

**Comment by danking** [comment]
each fix reveals deeper issues...

**Comment by danking** [comment]
forthcoming PR supersedes this one.

**Pull request closed without merging**